waku/node/v2/waku_types.nim

```diff
@@ -32,11 +34,29 @@ type
 
   QueryHandlerFunc* = proc(response: HistoryResponse) {.gcsafe, closure.}
 
+
+  Index* = object
+    ## This type contains the  description of an index used in the pagination of waku messages
+    digest*: string
+    timestamp*: string 
```

Comment:
I think string is not needed here and `uint32` could be used as this is probably going to be a unix timestamp?

Reply:
I will change the timestamp to unix timestamp (which is of type `float`)